schedule/sle-micro/yam/slem_installation_autoyast.yaml

```diff
@@ -0,0 +1,14 @@
+name:           sle_micro_autoyast
+description:    >
+  Maintainer: QE Yam <qe-yam at suse de>
+  SUSE Linux Enterprise Micro tests
+schedule:
+  - autoyast/prepare_profile
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/logs
+  - console/textinfo
```

Comment:
I have no idea why need this module.

Reply:
> I have no idea why need this module.

This module gets useful text-based information from the system and upload it as a log, I think the log is quite useful.

Reply:
In fact this is not big thing there, while for me it is obvious transactional_update is more important for SLE Micro test. Anyway, we can merge this to make it run on OSD. Later we can enhance this if needed.